packages/suite-data/src/message-system/config/config.v1.json

```diff
@@ -0,0 +1,74 @@
+{
+    "version": 1,
+    "timestamp": "2021-03-03T03:48:16+00:00",
+    "sequence": 1,
+    "actions": [
+        {
+            "conditions": [
+                {
+                    "duration": {
+                        "from": "2021-03-01T12:10:00.000Z",
+                        "to": "2022-01-31T12:10:00.000Z"
+                    },
+                    "os": {
+                        "macos": ["10.14", "10.18", "11"],
+                        "linux": "<20.04",
+                        "windows": "!",
+                        "android": "*",
+                        "ios": "13"
+                    },
+                    "environment": {
+                        "desktop": "<21.10",
+                        "mobile": "!",
+                        "web": "<22"
+                    },
+                    "browser": {
+                        "firefox": ["82", "83"],
+                        "chrome": "*",
+                        "chromium": "!"
+                    },
+                    "transport": {
+                        "bridge": ["2.0.30", "2.0.27"],
+                        "webusbplugin": "*"
+                    },
+                    "settings": [
+                        {
+                            "tor": true,
+                            "btc": true
+                        },
+                        {
+                            "tor": false,
+                            "ltc": true
+                        }
+                    ],
+                    "devices": [
+                        {
+                            "model": "1",
+                            "firmware": "1.9.4",
+                            "vendor": "trezor.io"
+                        }
+                    ]
+                }
+            ],
+            "message": {
+                "id": "0f3ec64d-c3e4-4787-8106-162f3ac14c34",
+                "priority": 10,
+                "dismissible": true,
+                "variant": "warning",
+                "category": "banner",
+                "content": {
+                    "en-GB": "New Trezor firmware is available!",
+                    "de-DE": "Neue Trezor Firmware ist verfügbar!"
+                },
+                "cta": {
+                    "action": "internal-link",
+                    "href": "firmware-index",
+                    "label": {
+                        "en-GB": "Update now",
+                        "de-DE": "Jetzt aktualisieren"
+                    }
+                }
+            }
+        }
+    ]
+}
```

Comment:
Before merging we need to clean this config up right? I mean it would be used otherwise, correct?

Reply:
Yes, definitely. I have a note in the first post.

Btw if you already have something what we want to put into the config, let's do it.

Reply:
@prusnak anything in our minds that we would like to push via the messaging system already? I have one in my mind:
- Bridge versions [28, 29, 30] on web & Windows.

Reply:
@tomasklim let's not block this PR because of this. We'll figure it out later!

Reply:
sure, I am leaving the config there, for now, to make it easier for PR reviewers. Before merging, I will replace it with an empty config.

```
{
    "version": 1,
    "timestamp": "2021-04-01T00:00:00+00:00",
    "sequence": 1,
    "actions": []
}
```